dags/blocksec_plugin/rex_bank_borrow.py

```diff
@@ -1,68 +1,19 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
+from blocksec_plugin.abis import BORROW_ABI
```

Comment:
Lets make this `REX_BANK_ABI` and include both `borrow` and `deposit`, the contract address for the bank is: 0xaD39F774A75C7673eE0c8Ca2A7b88454580D7F53